extension/conversion/remote/src/main/java/org/apache/polaris/extension/conversion/remote/RemoteTableConverter.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.extension.conversion.remote;
+
+import io.smallrye.common.annotation.Identifier;
+import jakarta.enterprise.context.RequestScoped;
+import java.util.Map;
+import java.util.Optional;
+import org.apache.polaris.service.conversion.TableConverter;
+import org.apache.polaris.service.types.GenericTable;
+
+@RequestScoped
+@Identifier("remote")
+public class RemoteTableConverter implements TableConverter {
+
+  // TODO constructor takes the hostname, etc.
+  public RemoteTableConverter() {}
+
+  // TODO call remote service
+  @Override
+  public Optional<GenericTable> convert(
+      GenericTable table, Map<String, String> storageCredentials, int requestedFreshnessSeconds) {
```

Comment:
If its only `GenericTable` we want to be able to convert from an Iceberg table to other formats right?